files/tasks/checkout_ogr.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- set_fact:
+    reverse_dir: "{{ project_dir }}/rev_project_cloned"
+
+- name: Install ansible
+  dnf:
+    name:
+    - ansible
+    state: present
+  become: true
+- name: Install flexmock
+  pip:
+    name: flexmock
+  become: true
+- name: Checkout ogr
+  git:
+    clone: yes
+    repo: https://github.com/packit-service/ogr.git
+    dest: "{{ reverse_dir }}"
+- name: install ogr dependencies via ansible playbooks
+  command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-install-requirements-pip.yaml
+  args:
+    chdir: "{{ reverse_dir }}"
+- name: install and run ogr tests via ansible playbooks
+  command: ansible-playbook -e "ansible_python_interpreter=/usr/bin/python3" -v -c local -i localhost, files/zuul-tests.yaml
```

Comment:
you can use include here instead (`ansible-doc include`)

Reply:
didn't realize you are using the cloned playbooks; you could still use import but since this works, let's go for it